wolfcrypt/src/ecc.c

```diff
@@ -297,6 +297,48 @@ ECC Curve Sizes:
 
 #endif
 
+#ifdef WOLFSSL_ECC_BLIND_K
```

Comment:
Please document `WOLFSSL_BLIND_PRIVATE_KEY` and `WOLFSSL_ECC_BLIND_K` at the top of ecc.c.

Reply:
fixed